closure/templates/closure_js_template_library.bzl

```diff
@@ -47,6 +47,16 @@ def _impl(ctx):
         for f in dep.closure_js_library.descriptors.to_list():
             args += ["--protoFileDescriptors=%s" % f.path]
             inputs.append(f)
+
+    # plugin_modules - add required jars to inputs and build the classpath.
+    plugin_classpath = []
+    for dep in ctx.attr.plugin_module_deps:
+        for jar in dep[JavaInfo].transitive_runtime_deps.to_list():
+            plugin_classpath.append(jar.path)
+            inputs.append(jar)
+    if plugin_classpath:
+        args.insert(0, "--main_advice_classpath="+':'.join(plugin_classpath))
```

Comment:
AAICT, `rules_closure` doesn‘t really work on Windows right now (at least we don‘t test it on Windows), but maybe use `ctx.configuration.host_path_seperator` (https://docs.bazel.build/versions/master/skylark/lib/configuration.html#host_path_separator ) so we don‘t introduce any new incompatibilities.

Reply:
Done